pkg/action/env.container.go

```diff
@@ -428,6 +441,12 @@ func (c *containerEnv) containerCreate(ctx context.Context, a *Action, opts *typ
 
 	// Create a container
 	actConf := a.ActionDef()
+
+	// Replace action command with entrypoint command if it set.
+	if c.entrypoint != "" {
+		actConf.Command = strings.Split(c.entrypoint, " ")
```

Comment:
That's not right.
There are 2 different definitions in container - command and entrypoint.
You shouldn't override command. Use entrypoint.

Reply:
ImplementeImplemented entrypoint, please check